helper-cli/src/main/kotlin/common/Utils.kt

```diff
@@ -32,6 +35,11 @@ import com.here.ort.model.yamlMapper
 
 import java.io.File
 
+/**
+ * Represents a mapping from a repository URLs to list of [PathExclude]s for the respective repository.
```

Comment:
"...from repository URLs to lists of..."

Reply:
done.